assets/js/lesson_3/hours_to_sec.js

```diff
@@ -0,0 +1,9 @@
+const hoursInput = +prompt('Enter an hour you want to convert into seconds');
+
+if (!isNaN(hoursInput)) {
+    const hoursToSec = (hoursInput * 60) *60;
+    alert(`You have ${hoursToSec} in ${hoursInput} hours`)
+} else (alert("You've entered invalid value"))
+
+
+// or another we can use "const hoursToSec = hoursInput * 3600;" in second line, I guess.
```

Comment:
You can, but your solution -- better. Better calculate after isNaN check, it's more safety)